shell/renderer/api/electron_api_context_bridge.cc

```diff
@@ -303,7 +322,9 @@ v8::MaybeLocal<v8::Value> PassValueToOtherContext(
   // Serializable objects
   blink::CloneableMessage ret;
   {
-    v8::Context::Scope source_context_scope(source_context);
+    v8::Context::Scope error_scope(error_target == BridgeErrorTarget::kSource
+                                       ? source_context
+                                       : destination_context);
```

Comment:
won't this result in the value produced from the converter being in a different context, as well as the error? if so, `error_scope` isn't the right name for this. additionally, i'm not sure that it's correct to produce the value in the destination context in this situation.

Reply:
No it won't, this conversion is **from** V8 and this context scope is only used for that conversion, the only V8 value that could get created during the conversion from V8 to `blink::Cloneable` is the error